### PR TITLE
[MIG] apps_download: Migration to 12.0

### PR DESCRIPTION
**Pull request by martintorresvm**
cc @Tecnativa

### CODE REVIEW

File: apps_download/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2017-2018 BizzAppDev
+# Copyright 2017 AgentERP
```
Comment (elicoidal) [inline apps_download/__manifest__.py]: Add Elico Corp
Dates -> 2019

File: apps_download/demo/product_template_demo.xml [hunk 2]
```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--Copyright (C) 2017-Today: BizzAppDev - Ruchir Shukla <ruchir@bizzappdev.com-->
```
Comment (elicoidal) [inline apps_download/demo/product_template_demo.xml]: make consistent with all authors

File: apps_download/demo/product_template_demo.xml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            <field name="summary">Summary Test</field>
+            <field name="website">Website Test</field>
+            <field name="description_rst">Description Test</field>
+            <field name="version">11.0</field>
+            <field name="author">OCA</field>
+            <field name="depends">base</field>
+            <field name="external_dependencies">{}</field>
+        </record>
+
+        <record id="product_product_100" model="product.product">
```
Comment (elicoidal) [inline apps_download/demo/product_template_demo.xml:66]: In this part the XML could be made more readable

**Review by elicoidal — COMMENTED** [review]
LGTM
some cleansing necessary on authors

File: apps_download/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply apps_download/__manifest__.py]: I don't see commit history for Elico. Do you mean yourself as reviewer? If so, it should go on contributors readme section, not here.

File: apps_download/readme/CONTRIBUTORS.rst [hunk 4]
```diff
@@ -0,0 +1,4 @@
+* Ruchir Shukla <ruchir@bizzappdev.com>
+* Georg Nottere <georg.notter@agenterp.com>
+* Eric Caudal <eric.caudal@elico-corp.com>
+* Victor M.M. Torres <victor.martin@elico-corp.com>
```
Comment (pedrobaeza) [inline apps_download/readme/CONTRIBUTORS.rst]: Incorrect e-mail (remembering old times, hehe), although we use the form:
```rst
* `Tecnativa <https://www.tecnativa.com>`_:

  * Víctor M.M. Torres
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: apps_download/__manifest__.py (hunk 1 above)
Comment (bizzappdev) [reply apps_download/__manifest__.py]: @elicoidal  was contributed functionality for deciding the functionality as well as testing it, and also review.

Comment (pedrobaeza) [reply apps_download/__manifest__.py]: OK, I see it's in authors and in contributors,so I think it's enough

Comment (elicoidal) [reply apps_download/__manifest__.py]: Yes (thanks @bizzappdev ): I did not contribute via PR but in the overall functional design of the APPS store via meeting and project management.
It is a bit unusual but I think I deserve authorship for all modules in this repository

**Comment by martintorresvm** [comment]
Changes done

File: apps_download/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply apps_download/__manifest__.py]: You already had authorship, but it's no so important the copyright line in this file, so it's done.

**Review by pedrobaeza — APPROVED** [review]

**Comment by martintorresvm** [comment]
Hope can be merged soon, please let me know by your updated review
cc @elicoidal

**Pull request merged**